features/step_definitions/logging.rb

```diff
@@ -1312,3 +1309,13 @@
     raise "Cluster doesn't have sufficient cpu or memory for logging pods to deploy, skip the logging case"
   end
 end
+
+Given /^I check all pods logs in the#{OPT_QUOTED} project$/ do | namespace |
+  ensure_admin_tagged
+  pods = BushSlicer::Pod.list(user: admin, project: project(namespace))
+  pods.each do | pod |
+    pod.containers.each do | container |
+      @result = admin.cli_exec(:logs, n: namespace,resource_name: pod.name, c: container.name)
```

Comment:
@anpingli I registered the step as a clean-up step in our scenario, so anytime when the case fails, the pods' logs will be recorded. In this condition, if checking the log content in this step, once there has any errors, the test run will be terminated.

Reply:
Let me try to add an parameter to this step to determine if it needs to check the content or not.

Reply:
yes. we have to escape  some messages.

Reply:
It's not easy to determine what kind of errors should be ignored and when the error should be ignored. Take above error as an example: if it happens when the ES pods start, it can be ignored, but if it happens after ES pods running for  a while, it shouldn't be ignored.

Reply:
Yes. Could we use   --since=0s. We can check only the error in x minutes.

Reply:
OK, I added steps to check logs in 5 minutes, since we have https://issues.redhat.com/browse/LOG-2674, the case may fails when testing logging 5.4